### DISCUSSION TITLE
Fan noise

### DISCUSSION THREAD

**Discussion by Babyte40** [Jyers/Marlin#2025] [General Discussion]
Hi, i have strange fan noise at 50% of speed, it's like unstable RPM, when it's going at 100% the noise is gone, maybe there is something to configure to fix it?

**Comment by UliLac** [comment]
First, which fan are you talking about?
Then - it might be some sort of resonance, the whereabouts of the fan cause some acoustic feedback dependent on revolutions. Some dampening might solve the problem, as might shifting of cables (for the MB fan).
